scipy/integrate/_quad_vec.py

```diff
@@ -144,36 +155,41 @@ def quad_vec(f, a, b, epsabs=1e-200, epsrel=1e-8, norm='2', cache_size=100e6, li
         Default: 'gk21' for finite intervals and 'gk15' for (semi-)infinite
     full_output : bool, optional
         Return an additional ``info`` dictionary.
+        
+        .. deprecated:: 1.11.0
+           Parameter `full_output` is deprecated and will be removed in version
+           1.13.0. When `full_output` is unspecified, the contents of ``info``
+           are provided as an attribute of the result object.
     args : tuple, optional
         Extra arguments to pass to function, if any.
 
         .. versionadded:: 1.8.0
 
     Returns
     -------
-    res : {float, array-like}
-        Estimate for the result
-    err : float
-        Error estimate for the result in the given norm
-    info : dict
-        Returned only when ``full_output=True``.
-        Info dictionary. Is an object with the attributes:
-
-            success : bool
-                Whether integration reached target precision.
-            status : int
-                Indicator for convergence, success (0),
-                failure (1), and failure due to rounding error (2).
-            neval : int
-                Number of function evaluations.
-            intervals : ndarray, shape (num_intervals, 2)
-                Start and end points of subdivision intervals.
-            integrals : ndarray, shape (num_intervals, ...)
-                Integral for each interval.
-                Note that at most ``cache_size`` values are recorded,
-                and the array may contains *nan* for missing items.
-            errors : ndarray, shape (num_intervals,)
-                Estimated integration error for each interval.
+    A result object with the following attributes:
+
+        res : {float, array-like}
```

Comment:
The object was just a tuple before, so I think it's reasonable to re-think the names. I called this first quantity `integral` in `nquad`, which I think is more descriptive than `res`. Whether `err` should be `abserr` as in `nquad` or if we should change it to `error` across the board is debatable. The rest of the names look OK to me.